hack/generate-manifest.sh

```diff
@@ -48,6 +48,7 @@ Generate a YAML manifest for Antrea using Helm and print it to stdout.
         --multicast                   Generates a manifest for multicast.
         --multicast-interfaces        Multicast interface names (default is empty)
         --extra-helm-values-file      Optional extra helm values file to override the default config values
+        --multicluster                Generates a manifest for multicluster.
```

Comment:
It also requires some extra change in antrea-agent's cluster role to watch `Gateway` and `ClusterInfoImport` events, it's defined in the file `build/charts/antrea/templates/agent/clusterrole.yaml` https://github.com/antrea-io/antrea/pull/3603/commits/d8a637bc93efe1b6d30e50e5e28f30ccf4f33614#diff-1db81cf91516a2a38ec22ed34f177e3d21564c7e19e12f76330a60466be136e6R189-R218

Reply:
OK, let me check with @tnqn , thanks.

Reply:
Yes, for all other cases, we add the permission specific to a feature by default, otherwise user not only need to toggle the configuration but also need to update RBAC to use a feature, which is not convenient.